### PR TITLE
[IMP] theme_bookstore: theme improvement

### PR DESCRIPTION
**Pull request by anso-odoo**
Revamp of `theme_bookstore`

Community:
- https://github.com/odoo/odoo/pull/176503

task-4109584

### CODE REVIEW

File: theme_bookstore/views/snippets/s_banner.xml [hunk 1]
```diff
@@ -7,8 +7,13 @@
         The best sellers <br/>of the year
     </xpath>
     <!-- Button -->
-    <xpath expr="//a[hasclass('btn')]" position="replace" mode="inner">
-        Discover it <span class="fa fa-angle-right ms-2"/>
+    <xpath expr="//a[hasclass('btn')]" position="replace">
+        <t t-if="request.env['ir.module.module'].sudo().search([('name', '=', 'website_sale'), ('state', '=', 'installed')], limit=1)">
+            <a href="/shop" class="btn btn-lg btn-primary"><i class="fa fa-shopping-cart me-2"/> Shop Now <span class="fa fa-angle-right ms-2"/></a>
```
Comment (stefanorigano) [inline theme_bookstore/views/snippets/s_banner.xml:12]: `role="img"`

File: theme_bookstore/views/snippets/s_banner.xml [hunk 2]
```diff
@@ -7,8 +7,13 @@
...
+        </t>
+        <t t-else="">
+            <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary">Start Now <span class="fa fa-angle-right ms-2"/></a>
```
Comment (stefanorigano) [inline theme_bookstore/views/snippets/s_banner.xml:15]: `role="img"`

**Review by stefanorigano — APPROVED** [review]
Some mirrors to address, could be done during the tech. review 👍 
LGTM @anso-odoo , good job.

File: theme_bookstore/views/snippets/s_showcase.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt80 pb80" remove="pt48 pb48"/>
+    </xpath>
+    <!-- Title -->
+    <xpath expr="//h3" position="replace" mode="inner">
+        We also sell prime quality second hand books
+    </xpath>
+    <!-- Lead -->
+    <xpath expr="//p[hasclass('lead')]" position="replace" mode="inner">
+        Prime quality second-hand books, carefully selected for value and charm— timeless stories at great prices.
```
Comment (sobo-odoo) [inline theme_bookstore/views/snippets/s_showcase.xml:15]: Is the "charm-" made like "I was about to say charming but i preferred to say timeless instead" or is there a typo ?

File: theme_bookstore/views/snippets/s_title.xml [hunk 4]
```diff
@@ -4,12 +4,12 @@
...
-        <attribute name="class" add="pt64 pb64 o_half_screen_height parallax s_parallax_is_fixed bg-black-50" remove="pt40 pb40" separator=" "/>
-        <attribute name="data-scroll-background-ratio">1</attribute>
+        <attribute name="class" add="pt120 pb120 o_cc o_cc5" remove="pt40 pb40" separator=" "/>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Airy/03_001','colors':{'c5':'#FC5B02'},'flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0','animated':'true'}</attribute>
     </xpath>
+    <!-- Shape -->
     <xpath expr="//div[hasclass('container')]" position="before">
-        <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 50%;"/>
-        <div class="o_we_bg_filter bg-black-50"/>
+        <div class="o_we_shape o_web_editor_Airy_03_001 o_we_animated" style="background-image: url('/web_editor/shape/web_editor%2FAiry%2F03_001.svg?c5=%23FC5B02');"/>
```
Comment (sobo-odoo) [inline theme_bookstore/views/snippets/s_title.xml:12]: There is no need in encoding with the `%2F` normally :slightly_smiling_face:

File: theme_bookstore/views/snippets/s_cta_box.xml [hunk 5]
```diff
@@ -20,7 +20,12 @@
     </xpath>
     <!-- Button -->
     <xpath expr="//a[hasclass('btn')]" position="replace">
-        <a t-att-href="cta_btn_href" class="btn btn-lg btn-primary"><span class="fa fa-shopping-cart"/> Shop Now</a>
+        <t t-if="request.env['ir.module.module'].sudo().search([('name', '=', 'website_sale'), ('state', '=', 'installed')], limit=1)">
+            <a href="/shop" class="btn btn-lg btn-secondary"><i class="fa fa-shopping-cart me-2" role="img"/>Shop Now<i class="fa fa-angle-right ms-2" role="img"/></a>
+        </t>
+        <t t-else="">
+            <a t-att-href="cta_btn_href" class="btn btn-lg btn-secondary">Start Now<i class="fa fa-angle-right ms-2" role="img"/></a>
+        </t>
```
Comment (sobo-odoo) [inline theme_bookstore/views/snippets/s_cta_box.xml:28]: Same comment about making a db request :grimacing:

**Review by sobo-odoo — CHANGES_REQUESTED** [review]
Here is my review :slightly_smiling_face:

**Comment by anso-odoo** [comment]
Thanks ! @sobo-odoo Everything should be fine now 👍

**Comment by sobo-odoo** [comment]
@anso-odoo @stefanorigano I [force pushed](https://github.com/odoo/design-themes/compare/f6ccaef44267a5df70ef8c8091e55d680f7cadce..2f29f8e422ab61d5894a44bec805e00a195c4bdd) to:
- replace the forced orange color of the "Title" shape by `o-color-1`
- remove the configurator versions of the "Call to action" and the "Image Text" snippets, since they are not dropped by the configurator anymore (there was a crash because of call to action) :+1: 
- remove the last step of the tour, since there is no `snippet[5]` anymore :+1:

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**